build_projects/dotnet-cli-build/project.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "1.0.0-*",
+  "description": "Build scripts for dotnet-cli",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "NETStandard.Library": "1.5.0-rc2-24027",
+    "Microsoft.CSharp": "4.0.1-rc2-24027",
+    "Microsoft.DotNet.Cli.Build.Framework": "1.0.0-*",
+    "System.Dynamic.Runtime": "4.0.11-rc2-24027",
+    "System.Reflection.Metadata": "1.3.0-rc2-24027",
+    "System.Runtime.Serialization.Primitives": "4.1.1-rc2-24027",
+    "System.Xml.XmlSerializer": "4.0.11-rc2-24027",
+    "WindowsAzure.Storage": "6.2.2-preview",
+
+    "shared-build-targets-utils": {"target": "project"}
```

Comment:
I think the way we project-to-project reference should be the same between here and the dependency on `Microsoft.DotNet.Cli.Build.Framework` above.